ldes-server-port-ingestion-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/ingestion/rest/exceptions/MalformedMemberIdException.java

```diff
@@ -2,13 +2,16 @@
 
 public class MalformedMemberIdException extends RuntimeException {
 
-	public MalformedMemberIdException() {
+	private String expectedMemberType;
+
+	public MalformedMemberIdException(String expectedMemberType) {
 		super();
+		this.expectedMemberType = expectedMemberType;
 	}
 
 	@Override
 	public String getMessage() {
-		return "Member id could not be extracted. MemberType could not be found in listStatements.";
+		return String.format("Member id could not be extracted. MemberType %s could not be found in listStatements.", expectedMemberType);
```

Comment:
In general, we avoid this kind of formatting, because it is less efficient than a simple concatenation.
"... MemberType" + expectedMemberType + "..."